extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ProcessAction.scala

```diff
@@ -12,13 +12,10 @@ import java.util.UUID
                                     processVersionId: VersionId,
                                     performedAt: Instant,
                                     user: String,
-                                    action: ProcessActionType,
+                                    actionType: ProcessActionType,
```

Comment:
Please mention about this change in migration guide - it is a part of the api

Reply:
Fixed.